src/amber/cli/templates/field.cr

```diff
@@ -17,6 +17,8 @@ module Amber::CLI
         time:      ["time", "Time", "TIMESTAMP"],
         timestamp: ["time", "Time", "TIMESTAMP"],
         password:  ["password", "String", "VARCHAR"],
+        ref:       ["ref", "Int64", "BIGINT"],
+        reference: ["ref", "Int64", "BIGINT"],
```

Comment:
Ahh good. Is `reference` the correct keywork or `references` plural?

Reply:
This is describing the type of field so it should not be plural since its a noun, not an action. It answers the question of what type the field is.  `This field is a reference`, not `This field is a references`.